### PR TITLE
[EAGLE-4917] CLI for Model upload

### PR DESCRIPTION
**Pull request by luv-bansal**
### What

- Clarifai CLI based on [Design Doc](https://clarifai.atlassian.net/wiki/spaces/DevX/pages/3897262124/Clarifai+Python+SDK+CLI+Design)
- Implemented `Model upload`, `Model test locally` and `Run model locally` in the CLI

#### Model Upload 
```bash
clarifai model upload --model_path <model_path>
```

#### Model Test Locally  
```bash
clarifai model test-locally --model_path <model_path>
```

#### Run Model Locally  
```bash
clarifai model run-locally --model_path <model_path>
```

### Why

* Improve User experience of using SDK

### How

*

### Tests

* Locally tested 

### Notes

*

### CODE REVIEW

**Review by zeiler — APPROVED** [review]
nice!

**Review by sanjaychelliah — COMMENTED** [review]
Added some comments. Otherwise looks good to me!

File: clarifai/cli/model.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    '--skip_dockerfile',
+    is_flag=True,
+    help=
+    'Flag to skip generating a dockerfile so that you can manually edit an already created dockerfile.',
+)
+def upload(model_path, download_checkpoints, skip_dockerfile):
+  """Upload a model to Clarifai."""
+  from clarifai.runners.models import model_upload
+
+  model_upload.main(model_path, download_checkpoints, skip_dockerfile)
```
[the remark this answers is not in the document]
Comment (luv-bansal) [reply clarifai/cli/model.py:33]: Yes, this prints the model upload status

**Review by sanjaychelliah — APPROVED** [review]
LGTM

**Pull request merged**